cloudformation/ecs-conex.template.js

```diff
@@ -71,11 +71,6 @@ var conex = {
       Description: 'An email address to subscribe to alarms',
       Type: 'String',
       Default: 'devnull@mapbox.com'
-    },
-    LogAggregationFunction: {
```

Comment:
Do we need to remove this? We don't use it at mapbox right now, but removing it for this upgrade makes it a major version change that I'd rather we avoid...